openbb_platform/providers/cboe/tests/test_cboe_fetchers.py

```diff
@@ -102,11 +102,28 @@ def test_cboe_available_indices_fetcher(credentials=test_credentials):
 @pytest.mark.record_http
 def test_cboe_options_chains_fetcher(credentials=test_credentials):
     """Test Cboe options chains fetcher."""
+    # pylint: disable=import-outside-toplevel
+    import asyncio
+
     params = {"symbol": "AAPL", "use_cache": False}
 
     fetcher = CboeOptionsChainsFetcher()
-    result = fetcher.test(params, credentials)
-    assert result is None
+    result = asyncio.run(fetcher.fetch_data(params, credentials))
+    result = result.result  # type: ignore
+    test_result = fetcher.test(params, credentials)
+    list_msg = "Unexpected data format, expected List"
+    oi_msg = "Unexpected keys in total_oi property, expected ['total', 'expiration', 'strike']"
+    assert test_result is None
+    assert isinstance(result.expirations, list), list_msg  # type: ignore
+    assert isinstance(result.strikes, list), list_msg  # type: ignore
+    assert isinstance(result.contract_symbol, list), list_msg  # type: ignore
```

Comment:
Why are we testing this here instead on the integration tests? We only want to ensure here that the TET process is running correctly and that the type hinting follows what is defined which the `test` method covers already.

Reply:
The main rationale for putting them in the unit tests:
  - methods are not accessible from the API.
  - the methods are directly attached to the response from the Provider Interface and can be used independently, Platform session not required to use these.
  - these tests check some of the validity of the data captured in the test cassette, if bad data is captured it will be exposed here.
  - these tests are run by the CI and GitHub actions, and can be used to ensure that all providers conform to the standard model.

Reply:
The main rationale for putting them in the unit tests:
  - methods are not accessible from the API.
  - the methods are directly attached to the response from the Provider Interface and can be used independently, Platform session not required to use these.
  - these tests check some of the validity of the data captured in the test cassette, if bad data is captured it will be exposed here.
  - these tests are run by the CI and GitHub actions, and can be used to ensure that all providers conform to the standard model.

Reply:
@deeleeramone let's stick to how we envisioned fetcher unit tests to run. Pydantic also ensures that the result is by what is defined. Moreover, if you want to check the `Property` output, you can create a new file/unit test that does just that with some mock data. This currently with how it is done is more like an integration test.

Reply:
Better?